nginx/ng/init.sls

```diff
@@ -6,6 +6,7 @@ include:
   - nginx.ng.config
   - nginx.ng.service
   - nginx.ng.vhosts
+  - nginx.ng.passenger
```

Comment:
It's probably better to just list this optional state in the README instead of keying it off a pillar setting.